src/app/modules/browse/browse-controller.js

```diff
@@ -83,23 +83,38 @@
                 params.bookOrBooks = _localize('book');
             }
 
+            var isFiltered = false;
+
             if (count === 0) {
-                message = _localize('There are no books that match your search for {shelf} {language} books', params);
+                $scope.resultsMessage = _localize('There are no books that match your search', params);
```

Comment:
Thanks for that analysis. I'm gonna decide in favor of clarity for the user for now, and ask you to revert that change.